stacker_blueprints/test/__init__.py

```diff
@@ -0,0 +1,125 @@
+import os.path
```

Comment:
Ok, I really like this - I think this would be better if it were included in stacker itself (maybe in https://github.com/cloudtools/stacker/blob/master/stacker/blueprints/testutil.py), and then we imported it here.  Also, it'd be awesome to have some docs for this there: http://stacker.readthedocs.io/en/latest/blueprints.html#testing-blueprints

What do you think?

Reply:
I guess I agree -- I'll move the code into stacker then, creating a new PR. I'll leave this open in the meantime.

Reply:
So awesome. Really looking forward to that!